tests/test_argument_check.py

```diff
@@ -29,14 +29,15 @@ def create_settings(test_type, vcs_type):
         settings.TeamcityServer.user_id = "TeamCityUser"
         settings.TeamcityServer.passwd = "TeamCityPassword"
 
-    if vcs_type == "git":
-        settings.GitVcs.repo = "git://127.0.0.1"
-        settings.GitVcs.refspec = "master"
+    if vcs_type == "git" or vcs_type == "gerrit":
+        settings.GitVcs.repo = "ssh://user@127.0.0.1"
+        settings.GitVcs.refspec = "refs/changes/20/203040/7"
```

Comment:
This refspec looks like a real one. I'd like it to be more obviously made up (like, '123456' or '000000' as review number), or at least a comment like '  # not an actual refspec'